gating/update_dependencies/run

```diff
@@ -19,11 +21,9 @@ git submodule update --init --remote
 osa_dir="${WORKSPACE}/openstack-ansible"
 git clone "https://github.com/openstack/openstack-ansible" ${osa_dir}
 pushd ${osa_dir}
-  git checkout $osa_branch
+  git checkout ${OSA_RELEASE}
```

Comment:
this should be quoted `"${OSA_RELEASE}"`

Reply:
will update

Reply:
done.